src/ci/docker/host-x86_64/dist-i686-linux/Dockerfile

```diff
@@ -1,48 +1,38 @@
-# We need recent curl, OpenSSL and CA certificates, so we can download further
-# dependencies in the debian:6 image. We use an ubuntu 20.04 image download
-# those.
-FROM ubuntu:20.04
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends \
-        curl \
-        ca-certificates
-WORKDIR /tmp
-COPY host-x86_64/dist-x86_64-linux/download-openssl-curl.sh /tmp/
-RUN ./download-openssl-curl.sh
-
-# We use Debian 6 (glibc 2.11, kernel 2.6.32) as a common base for other
-# distros that still need Rust support: RHEL 6 (glibc 2.12, kernel 2.6.32) and
-# SLES 11 SP4 (glibc 2.11, kernel 3.0).
-FROM debian:6
+# We document platform support for minimum glibc 2.17 and kernel 3.2.
+# CentOS 7 has kernel 3.10, but that's not as important as matching the glibc version.
```

Comment:
I think phrasing here could make it clearer that the newer kernel on the host doesn't matter as long as rustc does not actually use newer kernel APIs (without fallback).

Reply:
Ok, I updated this comment, also emphasizing that it's about kernel _headers_, since the actual running kernel in CI is going to be something even newer.